Subject: [PATCH] t/regenerate_errno.h

	* sysdeps/mach/hurd/bits/errno.h: Regenerate.

---

The build system will regenerate the file itself, but keeping the in-tree file
up-to-date avoids having spurious rebuilds due to that.

 sysdeps/mach/hurd/bits/errno.h |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/sysdeps/mach/hurd/bits/errno.h b/sysdeps/mach/hurd/bits/errno.h
index a7d57ad..3b6fe76 100644
--- a/sysdeps/mach/hurd/bits/errno.h
+++ b/sysdeps/mach/hurd/bits/errno.h
@@ -212,8 +212,8 @@ enum __error_t_codes
 #define	EPROTO          _HURD_ERRNO (116)/* Protocol error */
 	ETIME           = _HURD_ERRNO (117),
 #define	ETIME           _HURD_ERRNO (117)/* Timer expired */
-	ECANCELED       = _HURD_ERRNO (118),
-#define	ECANCELED       _HURD_ERRNO (118)/* Operation canceled */
+	ECANCELED       = _HURD_ERRNO (119),
+#define	ECANCELED       _HURD_ERRNO (119)/* Operation canceled */
 
 	/* Errors from <mach/message.h>.  */
 	EMACH_SEND_IN_PROGRESS          = 0x10000001,
@@ -297,7 +297,7 @@ enum __error_t_codes
 
 };
 
-#define	_HURD_ERRNOS	119
+#define	_HURD_ERRNOS	120
 
 /* User-visible type of error codes.  It is ok to use `int' or
    `kern_return_t' for these, but with `error_t' the debugger prints
-- 
tg: (0234227..) t/regenerate_errno.h (depends on: baseline)
